pkg/controller/podautoscaler/horizontal.go

```diff
@@ -843,11 +843,13 @@ func (a *HorizontalController) reconcileAutoscaler(ctx context.Context, hpaShare
 			retErr = err
 		}
 
-		logger.V(4).Info("Proposing desired replicas",
+		logger.V(2).Info("Proposing desired replicas",
```

Comment:
Why do we need to increase the original log level? If we need to troubleshoot a problem, we should increase the level of the controller-manager log instead of changing the log level.

Reply:
changed back log level to 4, thanks!